### PR TITLE
Fix PHP8 Deprecated: Required parameter $A follows optional parameter 

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Should fix all "Required parameter $A follows optional parameter $B warnings.

### Manual testing scenarios (*)

1. https://github.com/OpenMage/magento-lts/pull/1391/checks?check_run_id=1688556992

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ADDISON74** [comment]
Why this commit was closed? It should be merged in OpenMage.

**Review by colinmollenhour — APPROVED** [review]

File: app/code/core/Mage/Adminhtml/Block/Widget/Container.php [hunk 1]
```diff
@@ -131,11 +131,11 @@ public function removeButton($id)
      * Update specified button property
      *
      * @param string $id
-     * @param string|null $key
+     * @param string $key
```
Comment (Flyingmana) [inline app/code/core/Mage/Adminhtml/Block/Widget/Container.php:134]: it still accepts null as a param for this

File: lib/3Dsecure/CentinelClient.php [hunk 2]
```diff
@@ -48,61 +48,63 @@ function getValue($name) {
...
 
-		function getRequestXml(){
-			$queryString = "<CardinalMPI>";
-			foreach ($this->request as $name => $value) {
-				$queryString = $queryString."<".($name).">".($value)."</".($name).">" ;
-			}
-			$queryString = $queryString."</CardinalMPI>";
-			return "cmpi_msg=".urlencode($queryString);
-		}
+        function getRequestXml()
```
Comment (Flyingmana) [inline lib/3Dsecure/CentinelClient.php:60]: unrelated style change

File: app/code/core/Mage/Adminhtml/Block/Widget/Container.php [hunk 3]
```diff
@@ -162,11 +162,11 @@ protected function _updateButton($id, $key=null, $data)
      * Public wrapper for protected _updateButton method
      *
      * @param string $id
-     * @param string|null $key
+     * @param string $key
```
Comment (Flyingmana) [inline app/code/core/Mage/Adminhtml/Block/Widget/Container.php:165]: it still accepts null as a param here

**Review by Flyingmana — APPROVED** [review]
